docs/docs/widgets/bounded-box.md

```diff
@@ -131,25 +135,36 @@ A Tooltip is often used to specify the extra information when the user hovers th
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
+
+Here is the updated version of the "Devices" section with the requested changes:
```

Comment:
Same here, you don't need to add this to the documentation. Please remove it.

Reply:
@RudraPratik30 I have made the required changes as per given instructions. please check it if everything is fine.

Reply:
@RudraPratik30 please check this PR and and let me know if any change to be made or everything is good to go for merging my PR.